luigi/task.py

```diff
@@ -363,7 +388,15 @@ def get_params(cls):
             param_obj = getattr(cls, param_name)
             if not isinstance(param_obj, Parameter):
                 continue
+            params.append((param_name, param_obj))
 
+        pep593_params = _get_pep539_annotations(cls)
+        for param_name, param_obj in pep593_params:
+            try:
+                default_val = getattr(cls, param_name)
+                param_obj._default = default_val
```

Comment:
I do want to verify that this is safe to do. Do I need to do `param_obj = copy.copy(param_obj)` first?

The reason for doing this is that the `luigi.Parameter(default=...)` option only exists because the default is taken up by the parameter itself. But with this new syntax you can set a default value directly.